src/runtime/vdom/vdom-render.ts

```diff
@@ -111,6 +111,16 @@ const createElm = (oldParentVNode: d.VNode, newParentVNode: d.VNode, childIndex:
       elm.classList.add((elm['s-si'] = scopeId));
     }
 
+    if (BUILD.shadowDom || BUILD.scoped) {
```

Comment:
Can we maybe add a comment here to document what this section is doing?

Reply:
done 🚀